src/Models/StreamAnalyticsJob.ts

```diff
@@ -270,6 +270,16 @@ export class StreamAnalyticsJob implements Component, Provisionable,
       throw new Error('Initialize Azure client failed.');
     }
 
+    // Stop Job

```

Comment:
As stop job may take some time, could you print a message in console to notify user?